src/Pages/Home.js

```diff
@@ -0,0 +1,11 @@
+import BookStoreContainer from '../Components/BookStoreContainer';
+import NavBar from '../Components/NavBar';
+
+const Home = () => (
+  <>
+    <NavBar />
```

Comment:
Something is wrong with your navigation. When I click on the categories link I'm redirected to a blank page. Kindly look into it and make sure that I can see the Categories component.

- I think it would be best if you rendered the `NavBar.js` component in `App.js` since it will be visible on every page, instead of having it on both the `Categories.js` and `Home.js` components.